integration-tests/environments/react-native/ios/Podfile

```diff
@@ -1,5 +1,5 @@
-require_relative '../node_modules/react-native/scripts/react_native_pods'
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../../../../node_modules/react-native/scripts/react_native_pods'
```

Comment:
Does it not install RN in the project folder?

Reply:
With npm workspaces common dependencies are hoisted to the highest level, in this case the root's `node_modules`.